content/doc/book/updating/linux.adoc

```diff
@@ -0,0 +1,102 @@
+---
+layout: section
+title: Linux
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 3
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+link:https://www.jenkins.io/doc/book/installing/linux/[Linux installations]
+should be upgraded using the same method as was used for the original installation
+This section will guide you through updating your Jenkins instance to the most recent version.
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Linux
+[unordered.stack]
+1) Stop your Jenkins Instance :: 
+[source,bash]
+```
+#Stop running Jenkins service using the following command:
+
+sudo su -
+service jenkins stop
+```
+2) Take Jenkins instance backup as a recovery option ::
+[source,bash]
+```
+#Take backup of Jenkins home directory. In my case my Jenkins home directory is /var/lib/jenkins.
+
+cd /var/lib
+tar -cvzf jenkins_date.tar.gz jenkins/
+mv jenkins_date.tar.gz $HOME
```

Comment:
Replace with a link to the backup page, because there are many different ways to perform a backup and they depend on the environment.  For example, if running Ubuntu 22.04 with OpenZFS file system, the best backup may be to take file system snapshot, not a tar file.  Likewise with OpenSUSE 15.4 with btrfs.

Should not be in a code block because we should not describe all the backup alternatives here.  Link them to the correct page.
```suggestion
See the link:/doc/book/system-administration/backing-up/[backup instructions] for alternatives.
```